### ISSUE TITLE
some questions about trained model?

### ISSUE THREAD

**Issue by xieyufei1993** [zlmzju/itn#1]
Does ic15_itn_resnet.caffemodel means train in the icdar2015 data and msra_itn_resnet.caffemodel means train in the MSRA data ? why not use icdar2015 and msra to trian a single model?
@zlmzju

**Comment by zlmzju** [comment]
Yes, you are right. The models are used for fair comparisons with other similar methods to prove the effectiveness of ITN .  You can use more extra training data to achieve better performance.

**Comment by xieyufei1993** [comment]
ok，thank you very much !

**Comment by xieyufei1993** [comment]
when i run the test.sh,I meet this error:
![image](https://user-images.githubusercontent.com/19235054/45662921-7f1c7680-bb36-11e8-94ee-aefb4cc8ed5d.png)
@zlmzju

**Comment by xieyufei1993** [comment]
when i run the test.sh i meet this error:
![image](https://user-images.githubusercontent.com/19235054/45688511-24117080-bb84-11e8-948d-8c2905292d5b.png)
look forwad to hearing from you
